source/operations/monitoring/collect-minio-metrics-using-prometheus.rst

```diff
@@ -32,6 +32,9 @@ The procedure on this page documents the following:
 
    - An :mc:`mc` installation on your local host configured to :ref:`access <alias>` the MinIO deployment
 
+   These instructions use :ref:`version 2 metrics. <minio-metrics-v2>`
```

Comment:
I'd pull this out of the prerequisite admonition.
Move this and the next line up to just before the admonition starts, I suggest.